charts/rancher-logging/0.2.5/charts/fluentd/charts/fluentd-linux/values.yaml

```diff
@@ -3,7 +3,7 @@ labels: {}
 
```

Comment:
@nickgerace @paynejacob we can't make a 0.2.5 in 2.6 if 0.2.4 is in 2.5. How are we going to release a new version in 2.5 ? Should be 0.3.0. Somehow we didn't do that for 2.5 either, as 0.2.3 is in 2.4, we just got lucky. 
This also isn't include in the "new charts in 2.6" issue (32294).
cc @rmweir since you were involved in original PR

Reply:
Couldn't we just increment for the next 2.5 release?

Reply:
I'm wondering if 0.2.5 for 2.6 and 0.2.6 for 2.5.x is a bad approach

Reply:
If this is 0.2.5 in 2.6, and a different 0.2.5 is in 2.5, how could the upgrade from one to the other when they upgrade rancher.

Reply:
Fair point: I was thinking that `dev-v2.5` and `dev-v2.6` would have the same versions for some reason.
Perhaps we should increment the minor version...

```
0.2.x -- 2.5.x and below
0.3.x -- 2.6.x and below
```

Reply:
@cbron should I redo this pr with 0.3.0 for 2.6 and backport to 2.5 with a patch bump? Ill just pull all the changes from 0.2.5 into 0.3.0

Reply:
Yes, for system-charts every new Rancher version should bump the minor version. For feature charts its the major version. 
Also have to keep in mind the "every chart version gets forwardported to newer branches" rule. That also disqualifies 0.2.5.

Reply:
Yes we should only be releasing 0.3.0 in 2.6.0, so just change the chart version in this PR. 
Any changes that need to go into 2.5 should get a backport issue/PR, that won't just be a chart copy.